### PR TITLE
Wdc 15

### PR DESCRIPTION
**Pull request by Michael-Futral**
### <h1> **Nav menu needs to shrink and change background color** </h1>

Updating of JS, CSS, and HTML to make navigation bar respond to scrolling from top. At top of page navigation bar features will remain as original in develop branch. However, when screen width is larger than 992px and when scrolling from top new features below will be implemented: 

<hr/>
<h3>&#128512; New Feature(s) Overview: </h3>

- Navbar will shrink slightly
- Navbar will change colors
- The brand name "Web Dev Coop" will shrink in font size
- Also added a transition time for the animation so it is not clunky
- When returning to top navigation bar will return to as original
<hr/>

<details>
<summary><b>&#9193; Code addition details:</b></summary>
<ol>
<li>#mainNav" id to index.html 'nav' element on line #31. </li>
<li>Addition of #mainNav syntax to attribute '@media only screen and (min-width: 992px)' in main.css</li>
<li>Addition of main.js event listener that adds 'navbar-shrink' class to elements with id='mainNav' when scrolling more than 0 on y axis. </li>
<li>Addition of scripts element at end of index.js to pull in main.js.</li>
</ol>
</details>

### CODE REVIEW

**Review by wojo1086 — CHANGES_REQUESTED** [review]
Revert all unnecessary spacing changes.

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+window.addEventListener('DOMContentLoaded', event => {
+
+    // Navbar shrink function
+    var navbarShrink = function () {
```
Comment (wojo1086) [inline js/main.js:4]: Use `let` not `var`

**Review by wojo1086 — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (Michael-Futral) [reply js/main.js]: Changed var to let.

Also rearranged the few sections in main.js as close as I could to remember to what you did in the meeting monday.

**Comment by Michael-Futral** [comment]
Prettier has been removed in index.html (doesn't look like it messed with main.css),  and moved  <scripts src="js/main.js defer> to the head from where it original was at on the bottom of the file.

**Pull request merged**